examples/AlchemicalLoopSoftening/AlchemicalLoopSoftening.py

```diff
@@ -71,46 +97,45 @@ def __init__(self, **kwargs):
         # Atom Selection using MDtraj
```

Comment:
Isn't there a simpler way to specify all of this using the MDTraj DSL?  Like
```python
t.top.select('protein and ((resid 403 to 483) or (resid 1052 to 1109))')
```
and just get the atom list from there?